docs/Tutorials/Tools/Setting-up-VSCode.md

```diff
@@ -0,0 +1,180 @@
+---
+icon: octicons/paste-16
+tags:
+  - VSCode
+  - Editor
+---
+
+What is the most important thing before writing quests?
+Correct! We need a good text editor to make our lives easier with shortcuts and syntax highlighting.
+In this tutorial you will be introduced to Visual Studio Code (VSCode), the most popular text editor.
+
+In this tutorial you will learn how to install and use it to your advantage!
+
+
+<div class="grid" markdown>
+!!! danger "Requirements"
+    * No further requirements.
+
+!!! example "Related Docs"
+    * [YAML](../Getting-Started/Basics/YAML.md)
+</div>
+
+
+## 1. Download and installing VSCode
+
+In this step we will install VSCode on your computer.
+
+_Step 1:_ [Download the right version of Visual Studio Code](https://code.visualstudio.com/download) for your operating system from their website. 
+for your operating system.
+
+---
+_Step 2:_ Click on the VSCodeUserSetup-xXX-X.XX.X.exe in your download folder. (You'll
+find your _downloads_ folder in your file explorer.)
+
+---
+_Step 3:_ Go through the installation steps. The default settings are sufficient for BetonQuest usage.
+If you completed the installation and have the setup is closed. Move on with _Step 4_.
+
+---
+_Step 4:_ Start Visual Studio Code by double-clicking it on your desktop or starting it through your searchbar.
```

Comment:
I think this is really hard sticked to windows. May we dont need that at all or just start VSCode is enougth